.github/workflows/dokken-validate.yml

```diff
@@ -13,8 +13,6 @@ jobs:
           - ubuntu18
           - ubuntu20
           - redhat8
-        suite:
-          - aws-parallelcluster-install
```

Comment:
Why did you remove these lines? 
Even if we don't need them now, we can keep them for when we'll have more suites to run in parallel, can't we?

Reply:
Since now it is not mandatory but we can keep it there like a place holder: I'm going to add the lines again.